Solution/source/Submenus/VehicleModShop.cpp

```diff
@@ -70,49 +70,118 @@ namespace sub
 
 	};
 
-	const std::vector<NamedVehiclePaint> PAINTS_WHEELS
+	std::vector<NamedVehiclePaint> PAINTS_CURRENT{
+
+	};
+
+	const std::vector<NamedVehiclePaint> PAINTS_STATIC
```

Comment:
What is this and why is it omitting so many paint colors?  
Also, 156 is Alloy (DefaultAlloyColor), not "default  
And 160 (MP100SPEC) is the standard pearl of 158 (Pure Gold) and 159 (Brushed Gold)